styles/test/test-framework.scss

```
@@ -968,3 +988,45 @@ $test_note_grouped: (
     }
   }
 }
+
+@include describe('Platform-specifics') {
+  @include it('allows conditional execution') {
+    @include assert {
+      @include output {
+        @include for_platform(enum('Platform:::PDF')) {
+          platform: pdf;
+          color: blue;
+        }
+        @include for_platform(enum('Platform:::REX_WEB')) {
+          platform: rex_web;
+          color: green;
+        }
+      } @include expect {
+        platform: pdf;
+        color: blue;
+      }
+    }
+  }
+  @include it('allows function-based platform branching') {
+    @include assert-equal(
```

```suggestion
    @include assert-equal(
      if_platform(enum('Platform:::REX_WEB'), "hello", "goodbye"),
      "goodbye"
    );
    @include assert-equal(
```

I did not see the else case being tested